src/Filesystem/README.md

```diff
@@ -0,0 +1,377 @@
+# Filesystem Service
```

Comment:
Thank you :) Yes that would be a nice addition.
I will add a TOC if I have some time left.